openstack/compute/v2/extensions/simpletenantusage/requests.go

```diff
@@ -0,0 +1,72 @@
+package simpletenantusage
+
+import (
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// Get returns simple tenant usage data about all tenants
+func Get(client *gophercloud.ServiceClient, opts GetOptsBuilder) pagination.Pager {
+	url := getURL(client)
+	if opts != nil {
+		query, err := opts.ToSimpleTenantUsageGetMap()
+		if err != nil {
+			return pagination.Pager{Err: err}
+		}
+		url += query
+	}
+	return pagination.NewPager(client, url, func(r pagination.PageResult) pagination.Page {
+		return SimpleTenantUsagePage{pagination.LinkedPageBase{PageResult: r}}
+	})
+}
+
+// GetTenant returns simple tenant usage data about a specific tenant
+func GetTenant(client *gophercloud.ServiceClient, tenantID string, opts GetOptsBuilder) pagination.Pager {
+	url := getTenantURL(client, tenantID)
+	if opts != nil {
+		query, err := opts.ToSimpleTenantUsageGetMap()
+		if err != nil {
+			return pagination.Pager{Err: err}
+		}
+		url += query
+	}
+	return pagination.NewPager(client, url, func(r pagination.PageResult) pagination.Page {
+		return SimpleSingleTenantUsagePage{pagination.SinglePageBase(r)}
+	})
+}
+
+// Options for fetching the tenant usage for a Tenant.
+// All int-values are pointers so they can be nil if they are not needed
+// you can use gopercloud.IntToPointer() for convenience
+type GetOpts struct {
+	// Set to 1 to get detailed (i.e. "server usages") tenant usage; 0 otherwise.
+	Detailed *bool `json:"detailed,omitempty"`
```

Comment:
The URL parameter expects `detailed=1` or `detailed=0`. `*bool` only supports `true/false/nil`. So either there needs to be an internal custom conversion of `true` to `1` or this needs to be set to a `int` with documentation explaining that only a value of 1 will yield a result.

Also, since this is a query string, you'll need to use `q` instead of `json`. This goes for all other fields in this struct.